package.json

```diff
@@ -58,8 +58,9 @@
   "homepage": "https://github.com/thepracticaldev/dev.to#readme",
   "devDependencies": {
     "@storybook/addon-actions": "3.4.11",
+    "@storybook/addon-knobs": "3.4.11",
     "@storybook/addon-links": "3.4.11",
-    "@storybook/addons": "3.4.12",
+    "@storybook/addons": "3.4.11",
```

Comment:
All Storybook related versions need to be the same or you get a [channel error](https://storybook.js.org/docs/basics/faq/#why-is-there-no-addons-channel).